deployment_samples/EdgeDataStore/Install/templates/PIEgress.json

```diff
@@ -0,0 +1,10 @@
+,{
+  "Id": "<egressID>",
+  "ExecutionPeriod": "00:00:10",
+  "NamespaceId": "<database>",
+  "StreamPrefix": "<prefix>",
+  "Endpoint": "<url>",
+  "UserName": "<id>",
+  "Password": "<password>",
+  "ValidateEndpointCertificate": false
+}
```

Comment:
I can't comment on the file as it's binary, but you included `/Install/templates/installation_files/EdgeDataStore_linux-x64.deb`, which definitely should not be checked in.